examples/dashboard/teams.tf

```diff
@@ -0,0 +1,4 @@
+resource "chronosphere_team" "t" {
```

Comment:
optional: could this be a symlink to ../teams/main.tf

Reply:
done